storage/InsertDestination.cpp

```diff
@@ -60,25 +60,28 @@ InsertDestination::InsertDestination(const CatalogRelationSchema &relation,
                                      const StorageBlockLayout *layout,
                                      StorageManager *storage_manager,
                                      const std::size_t relational_op_index,
+                                     const std::size_t query_id,
```

Comment:
Ok -- how about in the sprit of moving forward, we close this discussion and PR if the reordering is the only issue. We can work on the reordering in a separate PR. 

@zuyu -- can you confirm that the reodering is the only issue that is left on the table from your perspective? Thanks.